sources/packages/web/src/components/students/applicationTracker/Assessment.vue

```diff
@@ -1,4 +1,9 @@
 <template>
+  <related-application-changed
+    v-if="
+      assessmentTriggerType === AssessmentTriggerType.RelatedApplicationChanged
+    "
+  />
```

Comment:
Minor. I know the existing code is not following the `-` separated format for the attributes of a component.  

```html
<assessment
        v-else-if="applicationStatus === ApplicationStatus.Assessment"
        :assessment-trigger-type="
          applicationProgressDetails.assessmentTriggerType!
        "
      />
``